test/index.js

```diff
@@ -74,24 +74,9 @@ describe('createClient', function(){
     assert.equal(client.amazon, 'http://www.google.com');
   });
 
-  it('should require a key', function(){
-    try {
-      ses.createClient();
-    } catch (err) {
-      if (!/key is required/.test(err)) {
-        throw err;
-      }
```

Comment:
FYI these tests didn't fail, even when I removed the `expect` check. Because if `ses.createClient()` *doesn't* throw then the test passes regardless.